src/SFA.DAS.AANHub.Database/Tables/Documents.sql

```diff
@@ -1,12 +1,9 @@
 ﻿CREATE TABLE [dbo].[Documents]
 (
     [Id] BIGINT NOT NULL IDENTITY(1, 1) PRIMARY KEY,
-    [CreatedBy] UNIQUEIDENTIFIER NOT NULL, 
-    [Created] DATETIME NOT NULL, 
```

Comment:
We need a Member association here. So I would suggest that we Rename CreatedBy to MemberId and restore foreign key

Reply:
Updated as requested